clang/lib/CodeGen/CodeGenModule.cpp

```diff
@@ -3975,9 +3975,9 @@ void CodeGenModule::generateIntelFPGAAnnotation(
     llvm::APSInt BWAInt = BWA->getValue()->EvaluateKnownConstInt(getContext());
     Out << '{' << BWA->getSpelling() << ':' << BWAInt << '}';
   }
-  if (const auto *MCA = D->getAttr<IntelFPGAMaxPrivateCopiesAttr>()) {
+  if (const auto *MCA = D->getAttr<IntelFPGAPrivateCopiesAttr>()) {
     llvm::APSInt MCAInt = MCA->getValue()->EvaluateKnownConstInt(getContext());
-    Out << '{' << MCA->getSpelling() << ':' << MCAInt << '}';
+    Out << "{max_private_copies:" << MCAInt << '}';
```

Comment:
If this change is required, then:
```suggestion
    Out << "{private_copies:" << MCAInt << '}';
```

Reply:
Ah, I got it. In that way we don't want to change LLVM IR output. It's looking hacky, isn't it? :)

Reply:
Exactly.. As you mentioned, the "max" here is redundant, and is in fact misleading to customers. That's why we're getting rid of it. However, I'm keeping the IR the same for now because otherwise, we would have to change the llvm-spirv translator and the SPIRV spec to do things the right way. I should open a JIRA to track that effort, but this does not affect the customer flow.

Reply:
>> However, I'm keeping the IR the same for now because otherwise, we would have to change the llvm-spirv translator and the SPIRV spec to do things the right way.

I'd argue that no changes would have to be made to the SPIR-V specification, as it doesn't say anything with regard to the LLVM IR layout. Would it be better to accompany this patch with a correspondent adjustment to the translator? I beliebe the fix would be trivial.

Reply:
@mohammadfawaz please address @AGindinson comment. I'm OK to go either way.

Reply:
@MrSidims @AGindinson I addressed the translator issue on the private repo. I will pull back here when I get approval there.